tasks/main.yml

```
@@ -23,6 +25,7 @@
 
 # Vhost configuration.
 - include: vhosts.yml
+  tags: nginx-config
```

what about
```suggestion
  tags: nginx-vhosts
```

`vhosts.yml` configure nginx virtual hosts and the task below configure nginx general configuration

---

I made it `nginx-config` since it also runs the task that configures `nginx.conf`